test/consider-listsize-directive.test.ts

```diff
@@ -0,0 +1,165 @@
+import { generateRandomQuery } from '../src/index'
+import { buildSchema, validate, print } from 'graphql'
+import * as dedent from 'dedent'
+
+test(`Ignore arguments when no @listSize is used`, () => {
+  const schema = buildSchema(`
+    directive @listSize(requireOneSlicingArgument: Boolean = true, assumedSize: Int, slicingArguments: [String], sizedFields: [String]) on FIELD_DEFINITION
+
+    type Order {
+      id: ID
+      date: String
+    }
+
+    type Query {
+      orders(first: Int, after: ID, last: Int, before: ID): [Order]
+    }
+  `)
+  const { queryDocument } = generateRandomQuery(schema, { seed: 1 })
+  const query = print(queryDocument)
+  expect(query.trim()).toEqual(
+    dedent(`query RandomQuery {
+      orders {
+        id
+        date
+      }
+    }
+    `).trim()
```

Comment:
Small comment but I think it is a bit odd that some of these strings start with a backtick, some of them start with a line break, some of them end with a backtick, and some of them end with a line break (and then a back tick).

Reply:
Will make them all start and end with a line break, as I think it is most readable.